ctapipe/image/concentration.py

```diff
@@ -0,0 +1,36 @@
+import numpy as np
+from ..io.containers import ConcentrationContainer
+
+
+def concentration(geom, image, hillas_parameters):
+    '''
+    Calculate concentraion values.
+
+    Concentrations are ratios of the amount of light in certain
+    areas to the full intensity of the image.
+
+    These features are usefull for g/h separation and energy estimation.
+    '''
+
+    h = hillas_parameters
+
+    delta_x = geom.pix_x - h.x
+    delta_y = geom.pix_y - h.y
+
+    # sort pixels by distance to cog
+    cog_pixels = np.argsort(delta_x**2 + delta_y**2)
+    conc_cog = np.sum(image[cog_pixels[:3]]) / h.intensity
```

Comment:
Why do you take the 3 closest pixels and more or less ? shouldn't we take the pixels where `cog_pixels <= r` where `r` is an argument of the method ?